src/System.Text.Json/src/System/Text/Json/Node/JsonNode.cs

```diff
@@ -2,6 +2,12 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+// for now disabling error caused by not adding documentation to methods
+#pragma warning disable CS1591
```

Comment:
It's easier to add them now. (Especially if only one Parse is supported in this PR).

Reply:
You mean add documentation? Yes, sure, I am planning to, I just didn't have time yet.